aspnetcore/mvc/views/razor.md

```diff
@@ -92,6 +100,20 @@ You can use an explicit expression to concatenate text with an expression result
 
 Without the explicit expression, `<p>Age@joe.Age</p>` is treated as an email address, and `<p>Age@joe.Age</p>` is rendered. When written as an explicit expression, `<p>Age33</p>` is rendered.
 
+You can also use explicit expressions to render output from generic methods in _.cshtml_ files.  In an implicit expression, the characters inside the brackets (`<>`) would be interpreted as an HTML tag:
+
+```cshtml
+<p>@GenericMethod<int>()</p>
+```
+
+The code generates a compiler error like `The "int" element was not closed.  All elements must be either self-closing or have a matching end tag` or `Cannot convert method group 'GenericMethod' to non-delegate type 'object'. Did you intend to invoke the method?`  However, when written as an explicit expression, the generic method's output will compile correctly.  
+
+```cshtml
+<p>@(GenericMethod<int>())</p>
+```
+
+Note that this restriction does not apply to _.vbhtml_ Razor files, because Visual Basic syntax places parentheses around generic type parameters instead of brackets.
```

Comment:
* does not :point_right: doesn't
*  \_.vbhtml\_ :point_right:  \*.vbhtml\*
* Drop comma: "files, because"

Reply:
👍